auth0/resource_auth0_connection_test.go

```diff
@@ -216,6 +212,7 @@ resource "auth0_connection" "ad" {
 	strategy = "ad"
 	show_as_button = true
 	options {
+		brute_force_protection = true
```

Comment:
The default by the API is to set brute force conection as enabled. Like this we remove the hack we introduced here https://github.com/auth0/terraform-provider-auth0/pull/97/files#diff-f2cec8e18d10bdff521bfda90a79b3154d267cccd5925b7ec9393cf7eeec7726L662-L666